followthemoney/types/name.py

```diff
@@ -1,7 +1,7 @@
 from banal import ensure_list, first
 from normality import slugify
 from normality.cleaning import collapse_spaces, strip_quotes
-from fuzzywuzzy import fuzz
+from fuzzywuzzy import fuzz  # type: ignore
```

Comment:
@pudo Can we just remove this, or is it being used elsewhere outside of these changes?

Reply:
We might want to ask @mynameisfiber about that. FtM used to use just `python-Levenshtein` but now we've adopted `fuzzywuzzy` which seems to wrap `python-Levenshtein`. I'm not quite sure what the value add is.

Reply:
It's being used in `NameType.compare()`. It uses `fuzz.WRatio` which has heuristics to apply differently weighted levenshtein-like distances depending on the size difference of the strings. It makes a huge difference when dealing with "Joe Shmo" vs "Joe Jonathan Shmo the Third" or other partial prefix/suffix matches.

Reply:
Thanks for confirming @mynameisfiber